build.gradle

```diff
@@ -306,6 +306,9 @@ shadowJar {
   relocate "org.checkerframework", "wiremock.org.checkerframework"
   relocate "org.hamcrest", "wiremock.org.hamcrest"
   relocate "org.slf4j", "wiremock.org.slf4j"
+  relocate "joptsimple", "wiremock.joptsimple"
+  relocate "javax.annotation", "wiremock.javax.annotation"
```

Comment:
I am not 100% sure we have enough test coverage to ensure it does not explode

Reply:
Thanks for the quick review and feedback, @oleg-nenashev! As an alternative, we could try excluding `javax.annotation` from the standalone artifact ...